src/sections/HomePortfolio/index.js

```diff
@@ -9,7 +9,6 @@ import withQuery from "./withQuery";
 
 import "./index.css";
 
-
```

Comment:
Do we need this new line?

Reply:
No, it's redundant. That was probably my linter that deleted the line. Should I revert it?

Reply:
yes please